yt_dlp/extractor/bibeltv.py

```diff
@@ -1,27 +1,202 @@
+
+        return formats, subtitles
+
+    @staticmethod
+    def _extract_base_info(data):
+        return {
+            'id': data['crn'],
+            'title': data.get('title'),
+            'description': data.get('description'),
+            'duration': int_or_none(data.get('duration'), scale=1000),
+            'timestamp': parse_iso8601(data.get('schedulingStart')),
+            'thumbnails': orderedSet(
+                traverse_obj(data, ('images', ..., {'url': ('url', {url_or_none})}))),
+            'season_number': data.get('seasonNumber'),
+            'episode_number': data.get('episodeNumber'),
+        }
+
+    def _extract_url_info(self, data):
+        return {
+            '_type': 'url',
+            'url': format_field(data, 'slug', 'https://www.bibeltv.de/mediathek/videos/%s'),
+            **self._extract_base_info(data),
+        }
+
+    def _extract_video_info(self, data):
+        video_id = data['id']
+        crn_id = data['crn']
+
+        json_data = self._download_json(
+            f'{self.API_URL}/video/{video_id}', crn_id, headers={'Authorization': self.AUTH_TOKEN})
+        if json_data.get('status') != 'success':
+            raise ExtractorError('Failed to load JSON metadata')
+
+        formats, subtitles = self._extract_formats_and_subtitles(
+            traverse_obj(json_data, ('video', 'videoUrls', ...)), crn_id)
+
+        return {
+            '_type': 'video',
+            **self._extract_base_info(data),
+            'formats': formats,
+            'subtitles': subtitles,
+            'view_count': data.get('viewCount'),
+            'like_count': data.get('likeCount'),
+        }
 
     def _real_extract(self, url):
         crn_id = self._match_id(url)
-        return self.url_result(
-            self.BRIGHTCOVE_URL_TEMPLATE % crn_id, 'BrightcoveNew')
+        video_data = traverse_obj(
+            self._search_nextjs_data(self._download_webpage(url, crn_id), crn_id),
+            ('props', 'pageProps', 'videoPageData', 'videos', 0, {dict}))
+        if not video_data:
+            raise ExtractorError('Missing video data.')
+
+        return self._extract_video_info(video_data)
+
+
+class BibelTvSerienIE(BibelTVIE):
+    IE_DESC = 'BibelTV'
+    _VALID_URL = r'https?://(?:www\.)?bibeltv\.de/mediathek/serien/(?:crn/)?(?P<id>\d+)'
+
+    IE_NAME = 'bibeltv:serien'
+
+    _TESTS = [{
+        'url': 'https://www.bibeltv.de/mediathek/serien/333485-ein-wunder-fuer-jeden-tag',
+        'playlist_mincount': 400,
+        'info_dict': {
+            'id': '333485',
+            'title': 'Ein Wunder für jeden Tag',
+            'description': 'Tägliche Kurzandacht mit Déborah Rosenkranz.',
+        },
+    }]
+
+    def _real_extract(self, url):
+        crn_id = self._match_id(url)
+        webpage = self._download_webpage(url, crn_id)
+        nextjs_data = self._search_nextjs_data(webpage, crn_id)
+        series_data = try_get(nextjs_data, lambda x: x['props']['pageProps']['seriePageData'], dict)
```

Comment:
```suggestion
        series_data = traverse_obj(nextjs_data, ('props', 'pageProps', 'seriePageData', {dict})) dict)
```
This also removes the `try_get` from imports.